sauna-app/pages/index.tsx

```diff
@@ -26,12 +34,22 @@ function SaunaPage(props: SaunaData) {
 
 
 //data fetching for pre-rendering
-export async function getStaticProps() {
-  const saunas = await fetchData()
+export async function getStaticProps(): Promise<SaunaProps> {
+  //OPTION 1 TO FETCH DATA:
+  //const saunas = await fetchData()
+  
+  //OPTION 2 TO FETCH DATA USING API
+  const saunas: SaunaData = await fetch('http://localhost:3000/api/saunas')
```

Comment:
A note about `async`/`await` and promise callbacks; don't mix them. I know these can feel confusing, and it takes time with them, so no worries 😊 

A quick explanation and how this can be solved:
1. You can use `async`/`await`:
```js
const saunasResponse: SaunaRespData = await fetch('/api/saunas')

const saunas: SaunasData = saunasResponse.data.saunas
```

or

```js
const saunas: SaunaData = await fetch('/api/saunas')
.then((response) => {
      return response.json();
    })
    .then((data) => {
      return data
    })
```

Reply:
I started digging deeper into this, and I'm super sorry for giving incorrect information before; so apparently, you can actually do what you did (= import the `loadSaunas`-function and use it in `getStaticProps`). But if you do that, I'd put the load function under `lib/load-saunas`  or something - not in the `api`-routes. 

Here's more information about that: https://nextjs.org/docs/basic-features/data-fetching/get-static-props#write-server-side-code-directly

So, you can forget that `localhost`-comment, and if you want, you can either merge this PR with the `localhost` and create another one to revert back to the `load`-function usage (and move it to `lib` (or utils or whatever you want to call it) -folder), or you can do it in this PR. Up to you 😊